edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/helpers/ContractDefinitionHelperFunctions.java

```diff
@@ -22,14 +22,14 @@
 import static org.eclipse.edc.spi.CoreConstants.EDC_NAMESPACE;
 
 public class ContractDefinitionHelperFunctions {
-    
+
     public static JsonObject createContractDefinition(String assetId, String definitionId, String accessPolicyId, String contractPolicyId) {
         return Json.createObjectBuilder()
                 .add(ID, definitionId)
                 .add(TYPE, EDC_NAMESPACE + "ContractDefinition")
                 .add(EDC_NAMESPACE + "accessPolicyId", accessPolicyId)
                 .add(EDC_NAMESPACE + "contractPolicyId", contractPolicyId)
-                .add(EDC_NAMESPACE + "criteria", Json.createArrayBuilder()
+                .add(EDC_NAMESPACE + "assetsSelector", Json.createArrayBuilder()
```

Comment:
shouldn't these be constants in EDC?

Reply:
There are but are in `ContractDefinitionRequestDto`, which require to pull that dep just for a couple of constants

Reply:
ok, fine since it's just a test